Content.Shared/_Impstation/Tools/SolutionRefuelSystem.cs

```diff
@@ -0,0 +1,87 @@
+using Content.Shared.Chemistry.Components;
+using Content.Shared.Chemistry.Components.SolutionManager;
+using Content.Shared.Database;
+using Content.Shared.DoAfter;
+using Content.Shared.Examine;
+using Content.Shared.FixedPoint;
+using Content.Shared.Interaction;
+using Content.Shared.Tools.Components;
+using Content.Shared.Whitelist;
+namespace Content.Shared.Tools.Systems;
```

Comment:
Using directive is unnecessary.
```suggestion
using Content.Shared.Chemistry.Components;
using Content.Shared.Chemistry.Components.SolutionManager;
using Content.Shared.Examine;
using Content.Shared.FixedPoint;
using Content.Shared.Interaction;
using Content.Shared.Tools.Components;

namespace Content.Shared.Tools.Systems;
```